web/src/routes/settings/debug/+page.svelte

```diff
@@ -2,11 +2,34 @@
     import { onMount } from "svelte";
     import { goto } from "$app/navigation";
 
+    import CopyIcon from "$components/misc/CopyIcon.svelte";
+
+    import { t } from "$lib/i18n/translations";
+    import { copyURL } from "$lib/download";
     import { version } from "$lib/version";
     import { device, app } from "$lib/device";
     import { defaultNavPage } from "$lib/subnav";
     import settings, { storedSettings } from "$lib/state/settings";
 
+    $: sections = [
+        { title: $t("settings.advanced.debug.device"), data: device },
```

Comment:
i don't think localization is needed for debug sections, it's better to leave them as-is